src/General/Badge/Badge.tsx

```diff
@@ -26,6 +33,7 @@ const Badge: React.FunctionComponent<Props> = ({
 interface Props extends React.ComponentPropsWithoutRef<typeof BadgeContainer> {
   label: string | number;
   sup?: boolean;
+  type?: string;
```

Comment:
should be `variant`?

Reply:
Yes yes, forgot to commit this file after changing the variable names to variant, from type.